.ci/release.sh

```diff
@@ -15,10 +38,13 @@ env DOWNLOAD_BASE="$DOWNLOAD_BASE" .ci/update.sh "$VERSION"
 .ci/update-to-production-downloads.sh
 
 # create a git patch file with the updates
-git checkout -b "update-$VERSION"
+UPDATE_BRANCH_NAME="update-$VERSION-$(date -u +%Y%m%d%H%M)"
+git checkout -b "$UPDATE_BRANCH_NAME"
 git add Formula
 git commit -m "Update to $VERSION" -m "Updated by homebrew-tap automation."
 mkdir -p .ci/output
-git format-patch -1 --output=".ci/output/update-$VERSION.patch"
+git format-patch -1 --stdout > ".ci/output/update-$VERSION.patch"
 git checkout -
 
+remove_homebrew
```

Comment:
Since these workers are reused, does it make sense to clean up at the end of the script, or would it be better to just do it at the beginning? I'm just trying to think of someone wanting to hop on the worker to debug the homebrew installation if things aren't working as expected.

Reply:
Only cleaning at the start sounds okay to me, done in https://github.com/elastic/homebrew-tap/pull/78/commits/2ab3a10d571c7105c37157df21e808f45d9e399e